--- a/drivers/kernelsu/sulog/event.c
+++ b/drivers/kernelsu/sulog/event.c
@@ -376,7 +376,7 @@ struct ksu_sulog_event* ksu_sulog_capture(const int event_type,
     u32 allowed = 0;
     gfp_t gfp = in_interrupt() ? GFP_ATOMIC : GFP_KERNEL;
 
-    pending = ksu_sulog_capture(KSU_SULOG_EVENT_IOCTL_GRANT_ROOT, NULL, USER_ARG_NULL, gfp);
+    pending = ksu_sulog_capture(KSU_SULOG_EVENT_IOCTL_GRANT_ROOT, NULL, *USER_ARG_NULL, gfp);
 
     if (IS_ERR_OR_NULL(pending)) {
         ksu_sulog_free(pending);
